### DISCUSSION TITLE
Encoding options

### DISCUSSION THREAD

**Discussion by OldGeezer916** [RandomEngy/VidCoder#972] [Q&A]
Just wondering about a couple boxes on the video encoding tab. What is 2-pass encoding & Turbo first pass? I guess they are checked by default because I had never really noticed them before. Do they have any affect on the speed or the quality of the encode? I convert everything to H.265, usually at 2400 kbbs. I have never touched the slider below that.

**Comment by RandomEngy** [comment]
2-pass does one pass to see what the video looks like (and which parts need more data applied to them). Then a second pass to actually do the encoding. It is slower, but better quality than using a set bitrate and a single pass.

However, I would recommend using quality targeting. That way you get the quality benefit of 2-pass but in only a single pass.

https://vidcoder.net/Documentation/VideoQualityOptions.html
